src/ui/Select.tsx

```diff
@@ -0,0 +1,31 @@
+import { Field } from 'formik';
+import styled from '@emotion/styled';
+import { FC } from 'react';
+
+const StyledField = styled(Field)`
+  background: var(--input-background);
+  border: 1px var(--text) solid;
+  color: var(--text);
+  height: 48px;
+  border-radius: 5px;
+  padding-left: 10px;
+  margin-bottom: 10px;
+`;
+
+interface SelectProps {
+  name: string;
+  options: Array<string | number>;
+  placeholder: string;
+}
+
+const Select: FC<SelectProps> = ({ name, options, placeholder }) => {
+  return (
+    <StyledField as="select" name={name}>
+      <option value="" label={placeholder} />
+      {options.map((option: string, i) => (
+        <option value={option} label={option} key={`${option}${i.toString()}`} />
```

Comment:
using indices as keys is an anti-pattern. the option itself should be unique. so `key={option}` should be fine